release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -553,6 +583,17 @@ In {op-system-base-full} 9, the default mode is cgroup v2. When {op-system-base-
 
 The Cluster Samples Operator is deprecated with the {product-title} 4.16 release. The Cluster Samples Operator will stop managing and providing support to the non-S2I samples (image streams and templates). No new templates, samples or non-Source-to-Image (Non-S2I) image streams will be added to the Cluster Samples Operator. However, the existing S2I builder image streams and templates will continue to receive updates until the Cluster Samples Operator is removed in a future release.
 
+[id="ocp-4-16-deprecation-operator-sdk_{context}"]
+==== Operator SDK and related Operators are deprecated
+
+The Operator SDK is deprecated with the {product-title} {product-version} release. Operator SDK is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. Additionally, the following related Operators are also deprecated and will be removed in a future release:
+
+* Ansible Operator
+* Go Operator
+* Helm Operator
+* Hybrid Helm Operator (Technology Preview)
+* Java Operator (Technology Preview)
```

Comment:
These aren't actually operators, they're operator base images that are used to write operators. We're not removing anything from the openshift runtime payload, and referring to them as `Ansible Operator`, `Helm Operator`, etc doesn't make much sense to me.

Also, as far as I know, there is no Go Operator or Java Operator base image. We just provide scaffolding tools for those, and then developers use a UBI or RHEL base image of some kind

Reply:
Addressed.